lib/board.rb

```diff
@@ -0,0 +1,64 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    sep = "-----------"
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts sep
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts sep
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(position)
+    position = position.to_i - 1
+    @cells[position]
+  end
+
+  def full?
+    @cells.all? { |position| position == "X" || position == "O" } ? true : false
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. 

`@cells.all? { |position| position == "X" || position == "O" }`